CHANGELOG.md

```diff
@@ -21,9 +21,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: Optimized the solver by allowing more resources as additional resources, allowing more actions to be skipped until the necessary resources are found.
 - Changed: For Multiworld, it's now preferred to have an additional pickups than placing it in another player's game, when there's no locations left in your game.
 - Changed: Randovania now internally uses the term `Region` for what used to be called a `World`. This is mostly an internal change.
+- Changed: Use the user's new discord public name instead of their username, for users that migrated to no longer have discriminators.
```

Comment:
"display name" is the official term for what we're talking about

Reply:
Oh, I forgot to check their doc for the proper name.

I got made after how this field I'm using is undocumented.
![image](https://github.com/randovania/randovania/assets/884928/442868e4-fd4b-4561-b35f-ddb928228156)
![image](https://github.com/randovania/randovania/assets/884928/6b7868e6-c9b3-43bc-9ff3-b08eb9d99560)